src/prommis/solvent_extraction/sx_model_flowsheet_steady_state.py

```diff
@@ -0,0 +1,155 @@
+""" 
+Demonstration flowsheet for steady state solvent extraction loading process
+using parameters and data derived from West Kentucky No. 13 coal refuse.
```

Comment:
Correct me if I'm mistaken, but isn't the plan for this flowsheet to represent the cleaner and rougher circuits (with recycles) as opposed to just a singular solvent extraction unit model in isolation?

Reply:
I am not sure, actually. This was the flowsheet I initially had, regarding the one which you mentioned, I asked you once about making this flowsheet, and you said you were making it at that time.

Reply:
Yes, but we ultimately decided that you should work on making this flowsheet since it'll be a good learning experience for you to develop a flowsheet with recycles. I can definitely help out if you have questions though

Reply:
Okay, I will start making it then, can I use the uky flowsheet as a reference for the recycle part?

Reply:
Sure

Reply:
@ArkoprabhoDasgupta It will probably make more sense to add this flowsheet in a separate PR since we'd like to get the documentation in before the end of the month